pkg/apiserver/registry/core/shoot/strategy.go

```diff
@@ -102,6 +104,9 @@ func mustIncreaseGeneration(oldShoot, newShoot *core.Shoot) bool {
 		var (
 			mustIncrease                  bool
 			mustRemoveOperationAnnotation bool
+			operations                    = v1beta1helper.GetShootGardenerOperations(newShoot.Annotations)
+			operationsSet                 = sets.New(operations...)
+			patchOperations               = operationsSet.UnsortedList()
```

Comment:
I was rather confused by `patchOperations`. How about `updatedOperations`or `remainingOperations` instead?

```suggestion
			remainingOperations               = operationsSet.UnsortedList()
```